src/components/NewProjectForm/index.js

```diff
@@ -0,0 +1,50 @@
+// @flow
+import React from 'react'
+import errorFinder from '@helpers/errorFinder'
+import Errors from '@components/Errors'
+import Wrapper from '@components/Wrapper'
+import Button from '@components/Button'
+import InputWithLabel from '@components/InputWithLabel'
+import SelectDropdown from '@components/SelectDropdown'
+import { type ValidationError } from '@types/ValidationError'
+import { Label } from './styles'
+
+type Props = {
+  onSubmit: () => any,
+  locales: Array<{ value: string, text: string }>,
+  errors: Array<ValidationError>
+}
+
+const NewProjectForm = ({ locales, onSubmit, errors }: Props) => {
+  const findErrors = errorFinder(errors)
+  const nameErrors = findErrors('name')
+
+  return (
+    <form onSubmit={onSubmit}>
+      <Wrapper>
+        <InputWithLabel
+          label="Project name"
+          name="name"
+          placeholder="MyAppName"
```

Comment:
Let’s add validations for the whole form. min/max matching the BE’s validations.